.cirrus.yml

```diff
@@ -679,6 +814,25 @@ rootless_remote_system_test_task:
         PRIV_NAME: rootless
 
 
+rootless_remote_system_test_aarch64_task:
+    matrix:
+        # Minimal sanity testing: only the latest Fedora
+        - env:
+              <<: *stdenvars_aarch64
+              # ID for re-use of build output
+              _BUILD_CACHE_HANDLE: ${FEDORA_NAME}-build-${CIRRUS_BUILD_ID}
+    <<: *local_system_test_task_aarch64
+    alias: rootless_remote_system_test_aarch64
+    depends_on:
+        - build_aarch64
+        - remote_integration_test
+    ec2_instance: *standard_build_ec2_aarch64
+    env:
+        TEST_FLAVOR: sys
+        PODBIN_NAME: remote
+        PRIV_NAME: rootless
+
+
```

Comment:
Arguable this one could probably be skipped for arm.

Reply:
skipping rootless system tests